index.html

```diff
@@ -181,6 +181,39 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+    var fireballSize = 22;
+    
+    var getFireballSpeed = function (left) {
+        if(left) {
+            return 5;
+        } else{
+            return 2;
+        }
+    };
+
+    var wizardSpeed = 3;
+    var wizardWidth = 70;
+
+    var getWizardHeight = function (WizardWidth) {
+        var WizardHeight = 1.337 * wizardWidth;
+        return WizardHeight;
+    };
+
+    function getWizardX(width) {
+        var WizardX = width / 2;
```

Comment:
Принято правило начинать начинать название переменных и функций с маленькой буквы

```suggestion
        var wizardX = width / 2;
```